rust_src/src/eval.rs

```diff
@@ -1,5 +1,7 @@
 //! Generic Lisp eval functions
 
+use libc::c_void;
```

Comment:
std is first, then other crates.

Reply:
Ah shoot, should have double checked. Fixed.